molgenis-app/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM tomcat:8.5-jre8
```

Comment:
Suggestion: use the alpine image here 8.5-jre8-alpine?

Reply:
I'm a bit worried how the c library in alpine combines with our python scripts, so I'd prefer to stick with the full image here which includes glibc.
We took a look and for docker image size, the difference is between 450Mb and 600Mb. If the main goal for now is developers pull request testing, my understanding is that that 150Mb difference only needs to be loaded once.

Ideally, in the end I'd like to end up with a jessie-slim based image. These are provided for higher level server jres.